### PR TITLE
Enable users to use a single tx to deploy a new customizable ERC20 token contract and make it tradable with Loopring Protocol

### PR DESCRIPTION
**Pull request by dong77**

### CODE REVIEW

**Comment by dong77** [comment]
Kongliang, there is a test failure with TokenRegistry.

**Comment by kongliangzhong** [comment]
@dong77 , test failure fixed.

File: migrations/2_deploy_registries.js [hunk 1]
```diff
@@ -1,9 +1,14 @@
...
 var NameRegistry            = artifacts.require("./NameRegistry");
+var TokenMint               = artifacts.require("./TokenMint");
 
 module.exports = function(deployer, network, accounts) {
-  deployer.deploy(TokenRegistry);
   deployer.deploy(TokenTransferDelegate);
   deployer.deploy(NameRegistry);
+
+  deployer.deploy(TokenMint).then(function() {
+    return deployer.deploy(TokenRegistry, TokenMint.address);
```
Comment (dong77) [inline migrations/2_deploy_registries.js]: Should we call TokenMint.initialize(...)?

**Comment by ramoslin02** [comment]
Great.

**Pull request merged**